docs/config/README.md

```diff
@@ -1179,6 +1180,37 @@ The module will be shown if any of the following conditions are met:
 symbol = "🐪 "
 ```
 
+## Perl
+
+The `perl` module shows the currently installed version of Perl.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `Makefile.PL` file
```

Comment:
There is also `Build.PL` used by [`Module::Build`](https://metacpan.org/pod/Module::Build) and [`Module::Build::Tiny`](https://metacpan.org/pod/Module::Build::Tiny).

Reply:
added, doc updated